StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFlowController.swift

```diff
@@ -308,6 +280,43 @@ extension PaymentSheet {
             }
         }
 
+        /// 🚧 Under construction
+        /// Call this method when the IntentConfiguration values you used to initialize PaymentSheet.FlowController (amount, currency, etc.) change.
+        /// This ensures the appropriate payment methods are displayed, etc.
+        /// - Parameter intentConfiguration: An updated IntentConfiguration
+        /// - Parameter completion: Called when the update completes with an optional error. Your implementation should get the customer's updated payment option by using the `paymentOption` property and update your UI. If an error occurred, retry. TODO(Update): Tell the merchant they need to disable the buy button.
+        /// Don’t call this method while PaymentSheet is being presented.
+        func update(intentConfiguration: IntentConfiguration, completion: @escaping (Error?) -> Void) {
+            // 1. Load the intent, payment methods, and link data from the Stripe API
+            PaymentSheet.load(
```

Comment:
I wonder if we can make this async/await so it would be easy to in-order updates without canceling or anything like that. I can explore that in a following PR but just a thought.